packages/reserve-site/public/assets/initial-target.svg

```diff
@@ -0,0 +1 @@
32 1.74.773 2.34 1.36.6.573 1.04 1.247 1.32 2.02.294.76.44 1.58.44 2.46a6.78 6.78 0 0 1-.54 2.7 7.26 7.26 0 0 1-1.5 2.24 7.02 7.02 0 0 1-2.22 1.5 6.78 6.78 0 0 1-2.7.54zm-.24-.7c1.04 0 1.96-.253 2.76-.76a5.09 5.09 0 0 0 1.86-2.1c.454-.893.68-1.92.68-3.08 0-.96-.16-1.813-.48-2.56-.32-.76-.746-1.393-1.28-1.9a5.47 5.47 0 0 0-1.8-1.18c-.666-.28-1.34-.42-2.02-.42l-1.48.06c-.4.027-.706.127-.92.3s-.32.46-.32.86v8.92c0 .533.147.933.44 1.2s.667.447 1.12.54c.467.08.947.12 1.44.12zm7.833.66a.95.95 0 0 1-.42-.08c-.107-.04-.16-.1-.16-.18 0-.093.053-.173.16-.24a2 2 0 0 1 .5-.22 3.25 3.25 0 0 0 .96-.48c.293-.213.547-.567.76-1.06l1.02-2.4.9-2.2.86-2.24.96-2.56.18-.56c.053-.187.087-.387.1-.6l.36-.12c.12-.04.24-.1.36-.18s.2-.2.24-.36c.053-.16.1-.267.14-.32.013-.013.027-.02.04-.02.027 0 .047.013.06.04.027.053.047.12.06.2l.08.24 1 3.1 1 3.06.9 2.64.66 1.88c.187.533.387.933.6 1.2s.433.453.66.56a3.56 3.56 0 0 0 .76.24c.227.04.393.107.5.2.107.08.16.167.16.26 0 .08-.053.133-.16.16s-.247.04-.42.04a9.5 9.5 0 0 1-.76-.04l-1.02-.06-1-.02h-.78l-.74.02h-.58c-.173 0-.313-.02-.42-.06s-.16-.1-.16-.18c0-.093.067-.173.2-.24a1.55 1.55 0 0 1 .46-.14c.44-.067.753-.18.94-.34s.28-.347.28-.56c0-.2-.047-.487-.14-.86l-.3-1.22-.34-1.16c-.027-.107-.08-.167-.16-.18-.067-.027-.193-.04-.38-.04h-3.78c-.053 0-.113.027-.18.08-.053.04-.107.133-.16.28l-.2.64-.28.9-.24.86c-.067.253-.1.413-.1.48 0 .293.113.54.34.74a2.05 2.05 0 0 0 .82.4c.44.12.66.253.66.4 0 .08-.053.14-.16.18-.107.027-.247.04-.42.04a21.38 21.38 0 0 1-.76-.02l-1.06-.02-1.28.06-1.12.06zm4.68-5.74h3.1c.187 0 .28-.033.28-.1 0-.027-.007-.053-.02-.08 0-.04-.007-.08-.02-.12l-1.56-4.88c-.08-.24-.14-.36-.18-.36-.027 0-.08.113-.16.34l-1.82 4.9c-.027.08-.04.133-.04.16a.11.11 0 0 0 .1.12c.08.013.187.02.32.02zm9.588 5.74c-.174 0-.314-.02-.42-.06-.107-.027-.16-.08-.16-.16 0-.16.22-.28.66-.36.506-.093.893-.247 1.16-.46s.4-.553.4-1.02V281c0-.467-.1-.8-.3-1-.187-.2-.534-.347-1.04-.44-.44-.08-.66-.213-.66-.4 0-.08.053-.133.16-.16.106-.04.246-.06.42-.06l.88.02.68.04h1.5l.6-.04.86-.02c.173 0 .313.02.42.06.106.027.16.08.16.16 0 .093-.054.173-.16.24s-.274.12-.5.16c-.494.093-.854.24-1.08.44s-.34.533-.34 1v9.04c0 .467.14.807.42 1.02s.666.367 1.16.46c.226.04.393.087.5.14s.16.127.16.22c0 .08-.054.133-.16.16-.107.04-.247.06-.42.06a8.3 8.3 0 0 1-.9-.04l-.66-.04-.78-.02-.88.02-.76.04c-.254.027-.56.04-.92.04zm7.397 4.18c.133 0 .246-.047.34-.14.28-.28.58-.68.9-1.2.333-.52.66-1.113.98-1.78s.606-1.373.86-2.12c.266-.76.473-1.513.62-2.26a10.5 10.5 0 0 0 .24-2.16c0-.853-.114-1.707-.34-2.56s-.52-1.66-.88-2.42c-.36-.773-.747-1.46-1.16-2.06-.4-.613-.78-1.1-1.14-1.46a.72.72 0 0 0-.26-.16.71.71 0 0 0-.28-.06c-.067 0-.16.033-.28.1-.107.053-.16.127-.16.22 0 .013.02.047.06.1s.08.113.12.18l.08.1c.4.467.8 1.093 1.2 1.88s.726 1.687.98 2.7c.266 1 .4 2.067.4 3.2 0 .853-.074 1.74-.22 2.66-.147.907-.34 1.787-.58 2.64a15.01 15.01 0 0 1-.82 2.24c-.294.667-.594 1.18-.9 1.54l-.1.1a1.54 1.54 0 0 1-.14.16c-.04.067-.06.107-.06.12a.38.38 0 0 0 .16.32c.12.08.246.12.38.12z" fill="#000"/></svg>
```

Comment:
This seems pretty large - maybe run through an optimizer?

Reply:
this is actually after the optimizer. it was twice as big before!